scss/modules/form.scss

```diff
@@ -40,7 +40,7 @@ input[type="text"] {
 input
 .input-description {
     margin-top: -5px;
-    font-size: 10px;
+    font-size: $smallFontSize;
```

Comment:
Doesn't this change the font size of a lot of different fields? Is that intended?